sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -0,0 +1,897 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.AI.MetricsAdvisor.Administration;
+using Azure.AI.MetricsAdvisor.Models;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.MetricsAdvisor.Tests
+{
+    public class DataFeedLiveTests : MetricsAdvisorLiveTestBase
+    {
+        public DataFeedLiveTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [RecordedTest]
+        public async Task CreateAndGetAzureApplicationInsightsDataFeedWithMinimumSetup()
+        {
+            MetricsAdvisorAdministrationClient adminClient = GetMetricsAdvisorAdministrationClient();
+
+            var dataFeedName = Recording.GenerateAlphaNumericId("dataFeed");
+            var dataSource = new AzureApplicationInsightsDataFeedSource("appId", "key", "cloud", "query");
+            DataFeed dataFeedToCreate = GetDataFeedWithMinimumSetup(dataFeedName, dataSource);
+
+            await using var disposableDataFeed = await DisposableDataFeed.CreateDataFeedAsync(adminClient, dataFeedToCreate);
+
+            DataFeed createdDataFeed = await adminClient.GetDataFeedAsync(disposableDataFeed.Id);
+
+            ValidateDataFeedWithMinimumSetup(createdDataFeed, disposableDataFeed.Id, dataFeedName);
+
+            Assert.That(createdDataFeed.SourceType, Is.EqualTo(DataFeedSourceType.AzureApplicationInsights));
+
+            var createdDataSource = createdDataFeed.DataSource as AzureApplicationInsightsDataFeedSource;
+
+            var expectedKey = TestEnvironment.Mode == RecordedTestMode.Playback ? "Sanitized" : "key";
```

Comment:
Unfortunately we can't check the authentication strings in Data Feed tests, since they're always sanitized during recording. This issue is only present in Playback, though.

Reply:
So to make sure I understand, the service doesn't check that it can connect to the specified DataFeeed, right? which is why u can pass whatever?
Can anyone access the `ApiKey` property? or the client needs to be created with a specific role or something for someone to get that value

Reply:
And the idea is not to suppress this sanitization in case someone uses a real key and then it gets logged?

Reply:
> So to make sure I understand, the service doesn't check that it can connect to the specified DataFeeed, right? which is why u can pass whatever?

Correct.

> Can anyone access the ApiKey property? or the client needs to be created with a specific role or something for someone to get that value

According to the service, it's only returned if you have the admin role for that Data Feed (which is always true for the person who created it).

> And the idea is not to suppress this sanitization in case someone uses a real key and then it gets logged?

Exactly.